### PR TITLE
Bump versions for 1.30.1 release

### PR DESCRIPTION
**Pull request by harupy**
## Related Issues/PRs

 #xxx

## What changes are proposed in this pull request?

As titled.

## How is this patch tested?

- [ ] Existing unit/integration tests
- [ ] New unit/integration tests
- [ ] Manual tests (describe details, including test results, below)

## Does this PR change the documentation?

- [ ] No. You can skip the rest of this section.
- [ ] Yes. Make sure the changed pages / sections render correctly in the documentation preview.

### Is this a user-facing change?

- [ ] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s), interfaces, languages, and integrations does this PR affect?

Components

- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/recipes`: Recipes, Recipe APIs, Recipe configs, Recipe Templates
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface

- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language

- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations

- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

<a name="release-note-category"></a>

### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [x] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

**Review by BenWilson2 — APPROVED** [review]
LGTM

**Comment by BenWilson2** [comment]
```
Error: The version '3.7.12' with architecture 'x64' was not found for Ubuntu 22.04.
```
I guess we have to use py 3.7.13...
From https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json -> 

  {
    "version": "3.7.13",
    "stable": true,
    "release_url": "https://github.com/actions/python-versions/releases/tag/3.7.13-2268493565",
    "files": [
      {
        "filename": "python-3.7.13-darwin-x64.tar.gz",
        "arch": "x64",
        "platform": "darwin",
        "download_url": "https://github.com/actions/python-versions/releases/download/3.7.13-2268493565/python-3.7.13-darwin-x64.tar.gz"
      },
      {
        "filename": "python-3.7.13-linux-18.04-x64.tar.gz",
        "arch": "x64",
        "platform": "linux",
        "platform_version": "18.04",
        "download_url": "https://github.com/actions/python-versions/releases/download/3.7.13-2268493565/python-3.7.13-linux-18.04-x64.tar.gz"
      },
      {
        "filename": "python-3.7.13-linux-20.04-x64.tar.gz",
        "arch": "x64",
        "platform": "linux",
        "platform_version": "20.04",
        "download_url": "https://github.com/actions/python-versions/releases/download/3.7.13-2268493565/python-3.7.13-linux-20.04-x64.tar.gz"
      },
      **_{
        "filename": "python-3.7.13-linux-22.04-x64.tar.gz",
        "arch": "x64",
        "platform": "linux",
        "platform_version": "22.04",
        "download_url": "https://github.com/actions/python-versions/releases/download/3.7.13-2268493565/python-3.7.13-linux-22.04-x64.tar.gz"
      }_**
    ]
  },
  {
    "version": "3.7.12",
    "stable": true,
    "release_url": "https://github.com/actions/python-versions/releases/tag/3.7.12-116024",
    "files": [
      {
        "filename": "python-3.7.12-darwin-x64.tar.gz",
        "arch": "x64",
        "platform": "darwin",
        "download_url": "https://github.com/actions/python-versions/releases/download/3.7.12-116024/python-3.7.12-darwin-x64.tar.gz"
      },
      {
        "filename": "python-3.7.12-linux-18.04-x64.tar.gz",
        "arch": "x64",
        "platform": "linux",
        "platform_version": "18.04",
        "download_url": "https://github.com/actions/python-versions/releases/download/3.7.12-116024/python-3.7.12-linux-18.04-x64.tar.gz"
      },
      {
        "filename": "python-3.7.12-linux-20.04-x64.tar.gz",
        "arch": "x64",
        "platform": "linux",
        "platform_version": "20.04",
        "download_url": "https://github.com/actions/python-versions/releases/download/3.7.12-116024/python-3.7.12-linux-20.04-x64.tar.gz"
      }
    ]
  },

File: .github/actions/setup-python/action.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ runs:
           if [ ${{ runner.os }} == "Windows" ]; then
             python_version="3.7.9"
           else
-            python_version="3.7.12"
+            python_version="3.7.13"
```
Comment (harupy) [inline .github/actions/setup-python/action.yml:30]:
```suggestion
            python_version="3.7.12"
```

**Comment by harupy** [comment]
@BenWilson2 Thanks, Ben! I merged the PR (assuming just bumping the versions normally doesn't break anything)

**Pull request merged**